tests/TestCase.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Tests;
+
+use Orchestra\Testbench\TestCase as BaseTestCase;
```

Comment:
Why are we using Orchestra rather than Phpunit test cases?

Reply:
It still uses PHPUnit under its hood but provides an easy way to start the Laravel container, just like how a standard Laravel app's `test/TestCase.php` has `CreatesApplication` trait to start the Laravel container.